wordpress-mobile/gutenberg-mobile
# Gutenberg hash for fixing crash related to pasting formatted text

Gutenberg: [#30676](https://github.com/WordPress/gutenberg/pull/30676)
Fixes [#30672](https://github.com/WordPress/gutenberg/issues/30672)

To test:
* Copy formatted text
* Paste formatted text, see no crash
* Paste formatted text without formatting, also see no crash.

PR submission checklist:
